datasets/fr/amf_regulatory_sanctions/fr_amf_regulatory_sanctions.yaml

```diff
@@ -0,0 +1,938 @@
+        entities:
+          - Consellior SAS
+      - match: 'La société Arkea Direct Bank'
+        entities:
+          - Arkea Direct Bank
+      - match: 'La société Global Derivative Trading GmbH<br /> M. Thorsten Wagner'
+        entities:
+          - Global Derivative Trading GmbH
+          - Thorsten Wagner
+      - match: 'La société Sud Conseils Patrimoine<br /> M. Patrice de Porrata-Doria'
+        entities:
+          - Sud Conseils Patrimoine
+          - Patrice de Porrata-Doria
+      - match: 'La société Corum Asset Management'
+        entities:
+          - Corum Asset Management
+      - match: 'La société Diana Holding<br /> La société DF Holding<br /> La société Compagnie Financière Européenne de Prises de Participation<br /> Madame Rita Zniber<br /> M. Serge Heringer'
+        entities:
+          - Diana Holding
+          - DF Holding
+          - Compagnie Financière Européenne de Prises de Participation
+          - Rita Zniber
+          - Serge Heringer
+      - match: 'Gestys SA <br /> M. Jean-Laurent Bruel'
+        entities:
+          - Gestys SA
+          - Jean-Laurent Bruel
+      - match: 'la société Efigest Conseil<br /> M. Jean-Noël Vignon<br /> M. Sylvain Bruley'
+        entities:
+          - Efigest Conseil
+          - Jean-Noël Vignon
+          - Sylvain Bruley
+      - match: 'La société Financière de Diane'
+        entities:
+          - Financière de Diane
+      - match: 'Sociétés Belvédère SA, Svi SNC, Sobieski SARL, des sociétés civiles Financière du Vignoble et Vermots Finance, de Mmes A et de'
+        entities:
+          - Belvédère SA
+          - Svi SNC
+          - Sobieski SARL
+          - Financière du Vignoble
+          - Vermots Finance
+      - match: 'et sociétés Verneuil et Associés, Verneuil Participations, Foch Investissements, FD Conseils et Participations et Duc SA'
+        entities:
+          - Verneuil et Associés
+          - Verneuil Participations
+          - Foch Investissements
+          - FD Conseils et Participations
+          - Duc SA
+      - match: 'CRÉDIT AGRICOLE CORPORATE AND INVESTMENT BANK, NATIXIS et société'
+        entities:
+          - CRÉDIT AGRICOLE CORPORATE AND INVESTMENT BANK
+          - NATIXIS
+      - match: 'Société Avendis Capital SA, la société Accent tonique BV et la société Accent Grave BV'
+        entities:
+          - Avendis Capital SA
+          - Accent tonique BV
+          - Accent Grave BV
+      - match: 'Société Altivie Asset Management du groupe Altivie'
+        entities:
+          - Altivie Asset Management
+          - Altivie
+      - match: 'et Refco Securities SA'
+        entities:
+          - Refco Securities SA
+
+        relationships:
+          - schema: Succession
+            from_prop: predecessor
+            from: Caisses d'Epargne d'Ile de France Paris 'Ile de France Nord
+            to_prop: successor
+            to: Ile de France
+            role: venant aux droits
```

Comment:
so the idea is that these would be under the `match` that they relate to. That's a) because they relate to that data and will disappear if Refco isn't listed any more, and b) it makes maintenance easier when they're colocated.